SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTestFlowController.cpp

```diff
@@ -49,20 +49,14 @@ void ASpatialFunctionalTestFlowController::OnActorReady(bool bHasAuthority)
 	{
 		// Registration of authoritative flow controllers (server and client)
 		OwningTest->RegisterFlowController(this);
-
-		if (WorkerDefinition.Type == ESpatialFunctionalTestWorkerType::Server)
-		{
-			TrySetReadyToRunTest();
-		}
 	}
+
+	TrySetReadyToRunTest();
```

Comment:
This only needs to be called if the flow controller has authority, so should be moved inside the check above.

Reply:
Moved it.
Originally, I wanted to remove the clientOwnership trigger completely, but it didn't pan out.
I still think it should somehow be possible to make this simpler, but I think I would rather just refactor the entire framework.